e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -151,6 +156,82 @@ describeEE("formatting > whitelabel", () => {
       cy.get("body").should("have.css", "font-family", `"${font}", sans-serif`);
     });
   });
+
+  describe("Help link", () => {
+    beforeEach(() => {
+      cy.intercept("PUT", "/api/setting/help-link").as("putHelpLink");
+      cy.intercept("PUT", "/api/setting/help-link-custom-destination").as(
+        "getHelpLinkUrl",
+      );
+    });
+
+    it("should allow customising the help link", () => {
```

Comment:
Good point, that was actually not working because the `:default` set in clj is only for the public setting, not the one available in the admin settings page.

Added the check here: https://github.com/metabase/metabase/pull/35960/files#diff-91071e9b59b8a7c88b8356cccf71e8eeff288d73fd1f78c4d5ea2d8904477ed0R175